notebooks/02_analyses/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>Can you add a sentence about what does the sanity check on the above sentence tell about the package/model? It seems to me that the above is a neutral sentence being classified as negative. does it make sense to explore other open source packages?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>It might make more sense to find a new NLP package that has a bit better results...for now I just addressed the model issues in the prose below.</p>